python-squid/src/main/java/org/sonar/python/metrics/FileLinesVisitor.java

```diff
@@ -99,8 +112,36 @@ public void leaveFile(AstNode astNode) {
 
     allLinesOfCode.put(inputFile, linesOfCode);
 
+    getContext().peekSourceCode().add(PythonMetric.COMMENT_LINES, linesOfComments.size());
+
     linesOfCode = Sets.newHashSet();
     linesOfComments = Sets.newHashSet();
   }
 
+  private Set<Integer> getLinesOfDocstring() {
+    DocstringVisitor<Grammar> docstringVisitor = new DocstringVisitor<>();
+    PythonAstScanner.scanSingleFile(getContext().getFile().getPath(), docstringVisitor);
+    Collection<AstNode> docstrings = docstringVisitor.getDocstrings().values();
+
+    Set<Integer> linesOfDocstring = new HashSet<>();
+    for (AstNode docstring : docstrings) {
+      if (docstring != null) {
+        TokenLocation location = new TokenLocation(docstring.getToken());
+        for (int line = location.startLine(); line <= location.endLine(); line++) {
+          linesOfDocstring.add(line);
+        }
+      }
+    }
+
+    return linesOfDocstring;
+  }
+
+  private void correctLinesOfCodeAndLineOfComments(Set<Integer> linesOfCode, Set<Integer> linesOfComments, Set<Integer> linesOfDosctring) {
+    for (Integer line : linesOfDosctring) {
+      linesOfCode.remove(line);
+      linesOfComments.add(line);
+      System.out.println("linesOfComments = " + linesOfComments);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106)